### PR TITLE
chore(icon): add deprecated warning for icon preload method

### PR DESCRIPTION
**Pull request by Nantawat-Poothong**
## Description
add a deprecated warning to icon preload method

Fixes # (issue)
https://jira.refinitiv.com/browse/STG-465

### CODE REVIEW

File: packages/elements/src/icon/utils/IconLoader.ts [hunk 1]
```diff
@@ -11,12 +12,14 @@ const iconLoaderInstance = new IconLoader();
 export { iconLoaderInstance as IconLoader };
 
 /**
+ * @deprecated Icon `preload` method is deprecated.
```
Comment (wattachai-lseg) [inline packages/elements/src/icon/utils/IconLoader.ts]:
```suggestion
 * @deprecated Icon `preload()` is deprecated.
```

File: packages/elements/src/icon/utils/IconLoader.ts [hunk 2]
```diff
@@ -11,12 +12,14 @@ const iconLoaderInstance = new IconLoader();
...
  * Helper function to preload set of icons.
  * It could help to reduce icon loading delay when ef-icon has a known set of icons that it can use.
  * @param attrs - list of arguments, representing icons.
  * Could be icon names, complete icon URLs or mix of both.
  * @returns Array of promises, which will be resolved with SVG bodies.
  */
 export const preload = (...attrs: string[]): Promise<string | undefined>[] => {
+  new DeprecationNotice('Icon `preload` method is deprecated.').once();
```
Comment (wattachai-lseg) [inline packages/elements/src/icon/utils/IconLoader.ts]: - let's use the same message with @deprecated of jsdoc
- `DeprecationNotice` should be created outside of `preload()` so the warning would display only once as intended.

**Review by wattachai-lseg — CHANGES_REQUESTED** [review]

File: packages/elements/src/icon/utils/IconLoader.ts [hunk 3]
```diff
@@ -12,14 +12,21 @@ const iconLoaderInstance = new IconLoader();
 export { iconLoaderInstance as IconLoader };
 
 /**
- * @deprecated Icon `preload` method is deprecated.
+   * Deprecation noticed, used to display a warning message
+   * when deprecated features are used.
```
Comment (wattachai-lseg) [inline packages/elements/src/icon/utils/IconLoader.ts]:
```suggestion
   * Deprecation notice displays a warning message
   * when deprecated features are used.
```

there probably are other locations with similar comment. Could you fix them as well?

Comment (Nantawat-Poothong) [reply packages/elements/src/icon/utils/IconLoader.ts]: Updated 😉👍

**Review by wattachai-lseg — APPROVED** [review]
🎉

**Pull request merged**